website/content/docs/concepts/architecture.md

```diff
@@ -2,7 +2,7 @@
 title: Heron Architecture
 ---
 
-Heron is the direct successor of [Apache Storm](http://storm.apache.org). From
+Heron provides compatibility with [Apache Storm](http://storm.apache.org). From
```

Comment:
Can we say more along the lines of
Heron is a general purpose streaming engine with performance, low latency, isolation, reliability in mind. It also provides API compatibilty with Storm.